src/hooks/usePoll.ts

```diff
@@ -60,7 +60,7 @@ export default function usePoll<T>(
     }
 
     async function poll(ttl = Date.now() + pollingInterval) {
-      timeout = setTimeout(poll, pollingInterval) // queue the next poll
+      timeout = window.setTimeout(poll, pollingInterval) // queue the next poll
```

Comment:
Why the `window.` addition?

Reply:
without explicitly setting window., `setTimeout` returns a Timeout object instead of the number we want

https://stackoverflow.com/questions/55550096/ts2322-type-timeout-is-not-assignable-to-type-number-when-running-unit-te

Reply:
ohhh I remember this. I think it might just be a TS issue in your editor?

using `window.` is fine!